jenkins-design-language/src/js/components/jtable/JTable.jsx

```diff
@@ -122,10 +122,12 @@ export class JTable extends Component {
                 newChildren.push(React.cloneElement(child, newChildProps));
             }
         });
-
+        // div element does not know anything about columns, to prevent warnings remove columns from props
+        if (restProps.columns) {
```

Comment:
No need for a test here, delete succeeds when the obj doesn't have that property (and the table should always have columns, also)